Projects/Team1/Semeniuk/FeatureSeleniumLogin/SeleniumTests/SeleniumTests/Class1.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeleniumTests
+{
```

Comment:
why do you need this class?

Reply:
Because firstly I launched the program, by a misclick, and after that visual studio told me to write the main function, because otherwise, I can do nothing. So I create another class(no test class).

Reply:
Іs it everything ok now?